app/src/interfaces/_system/system-theming/components/theme-color-picker.vue

```diff
@@ -0,0 +1,106 @@
+<template>
+	<div class="theme-color-picker" @click="activateColorPicker">
+		<input
+			ref="hiddenSourceInput"
+			:value="inputValue"
+			type="color"
```

Comment:
I am personally not a big fan of the native color picker in windows
![image](https://user-images.githubusercontent.com/9389634/172401422-8bce17e7-b2d9-4608-abbe-3d6cfded6e32.png)
For consistency across browsers and operating systems can we re-use the color picker logic already available in the app like the `Project settings > Project Color`? 
![image](https://user-images.githubusercontent.com/9389634/172402293-b60390ab-35e3-4919-9eef-f756e5a552c0.png)
 That also has the option to open the native color picker but prioritizes manual entry when clicking it.

Reply:
Lol TIL the windows color picker is fugly. That looks like it came straight from Windows 95!!

Reply:
It really has that look like it hasn't been updated since win 95 😂 (probably why chrome decided not to use the native picker at all on windows)

Reply:
I agree that the native color picker isn't terribly attractive. It looks even worse on windows 😂. However, there are a few issues we discussed when putting together the current interface.
- The design of the theming color selectors are too narrow to accommodate the dropdown of the existing color interface.
- With the number of fields on the page, we wanted to avoid messiness and information overload. Having a text input for each field just felt like too much.
- If I'm not mistaken, apart from the preset swatches, the existing color selector still uses the system color picker for color selection.

With that said, we did discuss the possibility of reworking the color picker interface. We briefly mentioned it here (and in Ben's reply): https://discord.com/channels/725371605378924594/957042592447623168/970326771297845358

Ultimately, what I'd like to do is create a custom color picker that can be triggered by a directive. Then we can reuse it between the color picker interface, and the theme color picker interfaces. For the sake of an MVP, we determined that this would be best added after finishing the initial theming system.

Reply:
To piggyback on this, I agree that we want to move forward with a custom picker... and the system options are less than ideal. A few points:
* We should standardize the custom color chooser component to be global and reusable
* The chooser should (like a dropdown) be able to be wider than the hit-zone target, so it can be used anywhere
* Again, it should be triggerable from anywhere...  including these interfaces, not only hex inputs
* It does allow for the system chooser, but that's an escape hatch deeper within the "pretty" chooser

While I am fine with the system chooser as a last resort, I'd love to see our color chooser reworked so it can be used here. It's up to the engineers if that should be MVP or not.